### ISSUE TITLE
Please consider using Fusion 360 for CAD

### ISSUE THREAD

**Issue by JetForMe** [openpnp/openpnp-openbuilds#5]
It's free and fully-functional, competitive with something like SolidWorks, and runs on both Mac and Windows. Thanks!

**Comment by vonnieda** [comment]
@JetForMe The next machine design will most likely use Fusion 360, but IMHO it's not worth doing all the conversion required to convert this one. Someone would need to go through and convert all the files, rebuild all the assemblies, create the constraints, etc.

**Comment by JetForMe** [comment]
Yeah, I understand. Sadly, I don't have Windows to use Cubify. Any chance you could export the files as STEP (or IGES)?

I can probably make do with the .stl files, or maybe I'll just design my head from scratch.

**Comment by vonnieda** [comment]
Unfortunately Cubify doesn't allow export of any useful file format, such
as STEP or IGES. That's one of the reasons this is such a problem. Hooray
for vendor lock in!

That being said, you can download free Windows VMs from here
https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/mac/

These run well in VMWare, VirtualBox, Parallels, etc. and combined with the
14 day free trial of Cubify that might be enough to get you going.

**Comment by JetForMe** [comment]
And I have to get a copy of Windows…it's all very tedious.

> On May 24, 2016, at 18:46 , Jason von Nieda notifications@github.com wrote:
> 
> Unfortunately Cubify doesn't allow export of any useful file format, such
> as STEP or IGES. That's one of the reasons this is such a problem. Hooray
> for vendor lock in!
> 
> That being said, you can download free Windows VMs from here
> https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/mac/
> 
> These run well in VMWare, VirtualBox, Parallels, etc. and combined with the
> 14 day free trial of Cubify that might be enough to get you going.
> 
> On Tue, May 24, 2016 at 6:43 PM Rick notifications@github.com wrote:
> 
> > Yeah, I understand. Sadly, I don't have Windows to use Cubify. Any chance
> > you could export the files as STEP (or IGES)?
> > 
> > I can probably make do with the .stl files, or maybe I'll just design my
> > head from scratch.
> > 
> > > On May 24, 2016, at 18:41 , Jason von Nieda notifications@github.com
> > > wrote:
> > > 
> > > @JetForMe The next machine design will most likely use Fusion 360, but
> > > IMHO it's not worth doing all the conversion required to convert this one.
> > > Someone would need to go through and convert all the files, rebuild all the
> > > assemblies, create the constraints, etc.
> > > 
> > > —
> > > You are receiving this because you were mentioned.
> > > Reply to this email directly or view it on GitHub
> > 
> > —
> > You are receiving this because you commented.
> > 
> > Reply to this email directly or view it on GitHub
> > https://github.com/openpnp/openpnp-openbuilds/issues/5#issuecomment-221451595
> > 
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly or view it on GitHub

## 

Rick Mann
rmann@latencyzero.com

**Comment by vonnieda** [comment]
Check that link in my previous message. They are free Windows VMs that run
out of the box. Just download, fire up your preferred VM host and you are
up and running with zero dollars spent. It's a pain, I agree, but when I
started this project Cubify was really the only reasonable option and now
I'm somewhat locked in with it.

**Comment by JetForMe** [comment]
No worries! I'll see how I get along, and if the pain gets high enough, I'll give it a shot.

What about dimensioned PDF files? I imagine that's a pain, too.

> On May 24, 2016, at 18:52 , Jason von Nieda notifications@github.com wrote:
> 
> Check that link in my previous message. They are free Windows VMs that run
> out of the box. Just download, fire up your preferred VM host and you are
> up and running with zero dollars spent. It's a pain, I agree, but when I
> started this project Cubify was really the only reasonable option and now
> I'm somewhat locked in with it.
> 
> On Tue, May 24, 2016 at 6:47 PM Rick notifications@github.com wrote:
> 
> > And I have to get a copy of Windows…it's all very tedious.
> > 
> > > On May 24, 2016, at 18:46 , Jason von Nieda notifications@github.com
> > > wrote:
> > > 
> > > Unfortunately Cubify doesn't allow export of any useful file format, such
> > > as STEP or IGES. That's one of the reasons this is such a problem. Hooray
> > > for vendor lock in!
> > > 
> > > That being said, you can download free Windows VMs from here
> > > https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/mac/
> > > 
> > > These run well in VMWare, VirtualBox, Parallels, etc. and combined with
> > > the
> > > 14 day free trial of Cubify that might be enough to get you going.
> > > 
> > > On Tue, May 24, 2016 at 6:43 PM Rick notifications@github.com wrote:
> > > 
> > > > Yeah, I understand. Sadly, I don't have Windows to use Cubify. Any
> > > > chance
> > > > you could export the files as STEP (or IGES)?
> > > > 
> > > > I can probably make do with the .stl files, or maybe I'll just design
> > > > my
> > > > head from scratch.
> > > > 
> > > > > On May 24, 2016, at 18:41 , Jason von Nieda <
> > > > > notifications@github.com>
> > > > > wrote:
> > > > > 
> > > > > @JetForMe The next machine design will most likely use Fusion 360,
> > > > > but
> > > > > IMHO it's not worth doing all the conversion required to convert this
> > > > > one.
> > > > > Someone would need to go through and convert all the files, rebuild
> > > > > all the
> > > > > assemblies, create the constraints, etc.
> > > > > 
> > > > > —
> > > > > You are receiving this because you were mentioned.
> > > > > Reply to this email directly or view it on GitHub
> > > > 
> > > > —
> > > > You are receiving this because you commented.
> > > > 
> > > > Reply to this email directly or view it on GitHub
> > > > <
> > > > https://github.com/openpnp/openpnp-openbuilds/issues/5#issuecomment-221451595
> > > > 
> > > > —
> > > > You are receiving this because you were mentioned.
> > > > Reply to this email directly or view it on GitHub
> > 
> > ## 
> > 
> > Rick Mann
> > rmann@latencyzero.com
> > 
> > —
> > You are receiving this because you commented.
> > 
> > Reply to this email directly or view it on GitHub
> > https://github.com/openpnp/openpnp-openbuilds/issues/5#issuecomment-221452136
> > 
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly or view it on GitHub

## 

Rick Mann
rmann@latencyzero.com

**Comment by vonnieda** [comment]
Cubify does let you make dimensioned drawings, but someone will have to do
all the work, and it's probably about as much work as just redrawing them
in Fusion. I'm happy to leave this issue open to see if someone is willing.

**Comment by Kenzu** [comment]
Hi

I know this is a old issue, but landed here via a Cubify google search. Just wanna tell that Cubify is based on Alibre Design/Atom, so it is possible to open all files in Alibre Design/Atom.
![billede](https://github.com/openpnp/openpnp-openbuilds/assets/505634/8d5c80df-c72e-4898-9e0c-e4fd20ad7cc9)

https://www.alibre.com

Don't know if this is of any use.

Jesper

Edit: export to STEP is possible in Alibre.

**Comment by vonnieda** [comment]
Hi @Kenzu  - thanks for posting this! I had noticed that Alibre was making some kind of comeback, and I was wondering if the files would still work. I'm happy to see they do!
